initrd/bin/kexec-insert-key

```diff
@@ -51,6 +51,8 @@ tpm extend -ix 4 -ic generic \
 
 # Check to continue
 if [ "$unseal_failed" = "y" ]; then
+	diff "$(dirname $INITRD)/kexec_lukshdr_hash.txt" /tmp/luksDump.txt \
+		&& echo "Headers of LUKSes to be unlocked via TPM do not change."
```

Comment:
Otherwise, tense of verb seems wrong, since they were previously measured.
"Headers of LUKSes to be unlocked via TPM do not change." -> "Headers of LUKSes to be unlocked via TPM did not change."

But as said in previous comment at line 54, it would be preferable for end user to know that it changed, not that it didn't change.